examples/trino-opa/trino/log.properties

```diff
@@ -0,0 +1,2 @@
+io.trino.server.security.oauth2=DEBUG
+io.trino=DEBUG
```

Comment:
wanna keep those on DEBUG?

Reply:
Helped me a lot to debug. Its not too noisy and I think it might help people to help themselves